### DISCUSSION TITLE
ilm prefix with ${aws:username}

### DISCUSSION THREAD

**Discussion by fdefilippo** [minio/minio#12034] [Q&A] [answered]
Hello, can I use the aws:username on ilm filter like this: 
```
    "Filter": {
     "And": {},
     "Prefix": "${aws:username}/out/"
    },
```
thank you.

**Comment by harshavardhana** [comment, marked as answer]
We do not support variable substitution for ILM, there is no explicit owner present that can be evaluated. So it's not possible to interpret such values without incoming authentication
